### ISSUE TITLE
high)  issue title  headings bookmarklet inaccessible to keyboard only users

### ISSUE THREAD

**Issue by KosiAsabere** [w3c/wai-easy-checks#215]
Location: Heading Structure - Easy Checks
*Example: "Check headings on this page" link

Issue:

 After activating the headings  bookmarklet  with the keyboard,  users navigating with a keyboard  may not be able to  find the modal until the end of the page. There is a  focus management issue. This can be disorienting and frustrating, especially for users relying on screen readers or those who  use a mouse. Modal does not correctly trap focus. modal also obscures texting in the background making it hard to tab to find the close button when zoomed in.

Suggested solution: by the more accessible option to test headings or perhaps at least a different example for the practice same page link.

Rationale:

When a modal opens, the focus should immediately move to the modal. The user shouldn't have to navigate through the entire page to find it.

Focus should be  Programmatically set to the modal or its first interactive element as soon as it opens. It will likely be easier to find a more accessible option for keyboard users. Like for example "accessibility insights"

**Comment by AndrewArch** [comment]
@iadawn the focus is immediately moved into the headings modal, but not kept there. In chrome I tabbed back into the page after the 'find out more' option.
